types/src/v0/impls/state.rs

```diff
@@ -745,7 +745,8 @@ fn validate_builder_fee(
         // TODO Marketplace signatures are placeholders for now. In
         // finished Marketplace signatures will cover the full
         // transaction.
-        if version.minor >= MarketplaceVersion::MINOR {
+        if version >= MarketplaceVersion::VERSION {
+            tracing::debug!("Validating sequencing fee signature.");
```

Comment:
```suggestion
            tracing::debug!("Validating (Marketplace) sequencing fee signature.");
```

Also add a log to the other branch of the if statement.